src/Microsoft.Identity.Client/Internal/Cache/BaseTokenCacheItem.cs

```diff
@@ -36,78 +36,32 @@ namespace Microsoft.Identity.Client.Internal.Cache
     [DataContract]
     internal abstract class BaseTokenCacheItem
     {
-        protected RequestContext RequestContext;
-
         /// <summary>
         /// Default constructor.
         /// </summary>
-        internal BaseTokenCacheItem(string authority, string clientId, TokenResponse response)
+        public BaseTokenCacheItem(string clientId)
```

Comment:
I noticed that you changed the visibility of a whole bunch of the internal class constructors, technically it doesn't matter for the visibility of internal class. Just out of curiosity of the reason of changing it.